src/styles/styles.ts

```diff
@@ -3886,6 +3890,15 @@ const styles = (theme: ThemeDefault) =>
             width: '100%',
         },
 
+        chatBottomLoader: {
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            height: CONST.CHAT_HEADER_LOADER_HEIGHT,
+        },
+
```

Comment:
```suggestion
```

This style is already defined as `boundaryLoaderStyles`

Reply:
👍🏻